samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: build.dev/v1alpha1
+kind: BuildStrategy
+metadata:
+  name: kaniko
+spec:
+  buildSteps:
+    - name: step-build-and-push
+      image: $(build.builderImage)
```

Comment:
We aren't going to support setting the kaniko image from Build CR, because we don't want it to overwritten.